extension/src/plots/model/collect.ts

```diff
@@ -156,12 +157,16 @@ const collectDatapoints = (
 ) => {
   for (const value of values) {
     const dvc_data_version_info = getDvcDataVersionInfo(value)
-    const data: { rev: string } = {
+    const data: { rev: string; dvc_data_version_info?: unknown } = {
       ...value,
       ...dvc_data_version_info,
       rev
     }
 
+    if (hasKey(data, 'dvc_data_version_info')) {
+      delete data.dvc_data_version_info
```

Comment:
Looking at the code, `revisionData` is only used for template plots, so should be safe to delete the key here. But please let me know if I missed something or if there is a better spot.